test/integration/runtime_int_test.go

```diff
@@ -117,8 +117,11 @@ func (suite *RuntimeIntegrationTestSuite) TestInUse() {
 	ts := e2e.New(suite.T(), false)
 	defer ts.Close()
 
-	cp := ts.Spawn("checkout", "ActiveState-CLI/Perl-5.32", ".")
-	cp.Expect("Skipping runtime setup")
+	cp := ts.SpawnWithOpts(
+		e2e.OptArgs("checkout", "ActiveState-CLI/Perl-5.36", "."),
+		e2e.OptAppendEnv(constants.DisableRuntime+"=false"),
```

Comment:
This shouldn't be needed because we're sourcing the runtime in the next step (`state activate`). I think you can revert this entire block except for the Perl-5.36 change.

Presumably this is leftover from previous debugging?